Source/Core/Core/Core.cpp

```diff
@@ -202,12 +202,12 @@ void DisplayMessage(std::string message, int time_in_ms)
 
 bool IsRunning(Core::System& system)
 {
-  return (GetState(system) != State::Uninitialized || s_hardware_initialized) && !s_is_stopping;
+  return s_state == State::Running;
 }
 
-bool IsRunningAndStarted()
+bool IsRunningOrStarting(Core::System& system)
 {
-  return s_is_started && !s_is_stopping;
+  return s_state == State::Running || s_state == State::Starting;
```

Comment:
Considering this is an atomic now, wouldn't this be better as:

```
const State state = s_state.load();
return state == State::Running || state == State::Starting;
```

Reply:
Done. I also added `.load()` to `IsRunning` and `IsUninitialized` just to make it more explicit to the reader that we are doing an atomic operation here.